src/explorer/SiteTreeItem.ts

```diff
@@ -167,6 +178,27 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
```

Comment:
I'm still a little confused by the naming because I generally think of the 'project' as the same folder as the 'workspace', but I assume `deployedProjectPath` is the full path being deployed? How about just `workspacePath` and `deployPath`

Reply:
Yeah, so the `activeWorkspacePath` is the active workspace that the user currently has opened vs. `deployedProjectPath` is the full path that is being deployed (the fsPath retrieved from the `appService.deploy`).  I'm fine with changing it though I'm not sure if it makes it clearer 😕